ReText/editor.py

```diff
@@ -350,11 +352,15 @@ def paintEvent(self, event):
 		cursor = QTextCursor(self.editor.document())
 		cursor.movePosition(QTextCursor.Start)
 		atEnd = False
+		if globalSettings.relativeLineNumbers:
+			relativeTo = self.editor.textCursor().blockNumber()
+		else:
+			relativeTo = -1
 		while not atEnd:
 			rect = self.editor.cursorRect(cursor)
 			block = cursor.block()
 			if block.isVisible():
-				number = str(cursor.blockNumber() + 1)
+				number = str(cursor.blockNumber() - relativeTo)
```

Comment:
Can you maybe use a proper typographic minus (&minus;) here? I.e. add `.replace('-', '−')` to this string?